### PR TITLE
Aerogear 9227 - Review supported configs (dev complete checkin)

### PR DESCRIPTION
**Pull request by jstaffor**
## Motivation
https://issues.jboss.org/browse/AEROGEAR-9227

## What
https://issues.jboss.org/browse/AEROGEAR-9227

## Why
https://issues.jboss.org/browse/AEROGEAR-9227

## How
https://issues.jboss.org/browse/AEROGEAR-9227

### CODE REVIEW

**Comment by wtrocki** [comment]
Looks good. I will hold of with merge until we confirm this stack by testing.

File: modules/ROOT/pages/supported_versions.adoc [hunk 1]
```diff
@@ -3,9 +3,10 @@ include::_partials/attributes.adoc[]
...
+As of July 2019, the following versions are tested and supported:
 
-* Cordova: version 8
-* Nodejs: version 8 and 10
-* Android SDK: API level 27
-* XCode: version 10.1
\ No newline at end of file
+* Cordova: version 9
+* Nodejs: version 10 and 12 LTS
+* Android SDK: API level 29
```
Comment (jhellar) [inline modules/ROOT/pages/supported_versions.adoc]: Should this be 28? From what I've looked, 29 was not released yet.

File: modules/ROOT/pages/supported_versions.adoc [hunk 2]
```diff
@@ -3,9 +3,10 @@ include::_partials/attributes.adoc[]
...
+Generally speaking, AeroGear Mobile Services aim to support the latest versions of tools and technologies.
+As of July 2019, the following versions are tested and supported:
 
-* Cordova: version 8
-* Nodejs: version 8 and 10
-* Android SDK: API level 27
-* XCode: version 10.1
\ No newline at end of file
+* Cordova: version 9
+* Nodejs: version 10 and 12 LTS
```
Comment (jhellar) [inline modules/ROOT/pages/supported_versions.adoc]: Version 10 is now LTS. But should we mention it all? Would just `version 10 and 12` be ok?

File: modules/ROOT/pages/supported_versions.adoc (hunk 1 above)
Comment (wtrocki) [reply modules/ROOT/pages/supported_versions.adoc]: I remember @danielpassos trying our stuff on top of the beta release. 
I wanted to save another bump but we can downgrade to 28

File: modules/ROOT/pages/supported_versions.adoc (hunk 2 above)
Comment (wtrocki) [reply modules/ROOT/pages/supported_versions.adoc]: good point. Let's mention both as LTS

File: modules/ROOT/pages/supported_versions.adoc (hunk 1 above)
Comment (danielpassos) [reply modules/ROOT/pages/supported_versions.adoc]: Well, Android 29 still in [preview version](https://developer.android.com/preview)

File: modules/ROOT/pages/supported_versions.adoc (hunk 2 above)
Comment (jstaffor) [reply modules/ROOT/pages/supported_versions.adoc]: Food for thought:
Node v**10.0.0** (Current) - https://nodejs.org/fr/blog/release/v10.0.0/
Node v**10.13.0** (**LTS**) - https://nodejs.org/fr/blog/release/v10.13.0/

In terms of accurately presenting info, I think we need to state "Node v10.13.0 (LTS)" if we want to mention LTS as it seems to me that "Node v10" isn't LTS. I'm open to correction on this as I'm not an expert in this area. What are your thoughts on this?

File: modules/ROOT/pages/supported_versions.adoc (hunk 1 above)
Comment (jstaffor) [reply modules/ROOT/pages/supported_versions.adoc]: I think it's safer to state "Android SDK: API level 28". Does any know what version(s) did we test this on/plan to test on?

Comment (jhellar) [reply modules/ROOT/pages/supported_versions.adoc]: Currently our integration tests run against API level 28, also device we are/will be testing on manually is 28.

Comment (wtrocki) [reply modules/ROOT/pages/supported_versions.adoc]: Agree! 28 is better for now

File: modules/ROOT/pages/supported_versions.adoc (hunk 2 above)
Comment (wtrocki) [reply modules/ROOT/pages/supported_versions.adoc]: We will always work and test on LTS

File: modules/ROOT/pages/supported_versions.adoc (hunk 1 above)
Comment (jstaffor) [reply modules/ROOT/pages/supported_versions.adoc]: Great - I'll make the udpate :+1:

File: modules/ROOT/pages/supported_versions.adoc (hunk 2 above)
Comment (jstaffor) [reply modules/ROOT/pages/supported_versions.adoc]:
```suggestion
* Nodejs: version Node v10.16.0 (LTS) and Node v12.x
```

Looking at the versions - there's no LTS for Node v12 hence the proposal - open to suggestion.

Comment (finp) [reply modules/ROOT/pages/supported_versions.adoc]: it's explained here https://endoflife.date/nodejs
v12 will become LTS in Oct (as far as I can see)

File: modules/ROOT/pages/supported_versions.adoc [hunk 3]
```diff
@@ -3,9 +3,10 @@ include::_partials/attributes.adoc[]
...
+As of July 2019, the following versions are tested and supported:
 
-* Cordova: version 8
-* Nodejs: version 8 and 10
-* Android SDK: API level 27
-* XCode: version 10.1
\ No newline at end of file
+* Cordova: version 9
+* Nodejs: version Node v10.16.0 (LTS) and Node v12.x
+* Android SDK: API level 29
```
Comment (jstaffor) [inline modules/ROOT/pages/supported_versions.adoc]:
```suggestion
* Android SDK: API level 28
```

**Comment by finp** [comment]
@wtrocki @jhellar  do you think we're good to merge this yet?

**Comment by wtrocki** [comment]
Yes

**Comment by finp** [comment]
@jhellar We can handle the Xcode version separately: My question for you and @wtrocki  is what happens if someone uses Xcode 10.2? do things stop working? 
Would it make sense to say Xcode 10.2 or higher?

**Comment by wtrocki** [comment]
@finp We should not care about this at all. Our relation to xcode comes with Cordova so let's say that we say that you should use Cordova 9 and this comes with specific support for android and IOS that has Xcode. We mentioning it for convenience so people do not need to go to cordova and check what versions they need. Generally, version range for support is very wide so I see this more as 
mentioning on what stack we are testing this.

**Pull request merged**